packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -99,17 +108,24 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
   // Eyeballed values. Feel free to tweak.
   static const Duration kFadeOutDuration = const Duration(milliseconds: 10);
   static const Duration kFadeInDuration = const Duration(milliseconds: 350);
+  Tween<double> _opacityTween;
 
   AnimationController _animationController;
 
+  Tween<double> get _tween => new Tween<double>(
```

Comment:
This creates an object so it should be a method rather than a getter.
But in general I would just have a void method that actually sets _opacityTween itself since that's common to both call sites too.

Also, avoid `=>` for multiline expressions.